vxwhatsapp/tests/test_claims.py

```diff
@@ -18,6 +19,7 @@ async def redis() -> AsyncGenerator[Redis, None]:
     await cleanup_redis()
 
 
+@pytest.mark.asyncio
```

Comment:
Happy with these, but it also seems like we can set the `asyncio_mode` to `auto`, which will make all tests with an `async def`, and all fixtures with `async def`, use `pytest_asyncio` automatically.

Reply:
I'll merge as-is and we can switch to auto later if we want. :-)